starters/gatsby-wordpress-starter/src/components/grid.jsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react'
 
-import { GatsbyImage, StaticImage } from 'gatsby-plugin-image'
+import { GatsbyImage } from 'gatsby-plugin-image'
 import { Link } from 'gatsby'
 
+const GradientPlaceholder = () => (
+	<div className="w-full h-full bg-gradient-to-b from-blue-400 to-blue-500" />
```

Comment:
🤔 I am wondering if the gradient is too subtle? maybe `from-blue-100 to-blue-500`?

@backlineint, what do you think?
Current:
![image](https://user-images.githubusercontent.com/63015754/191614558-e16dcaab-5cf1-49ac-b1b2-41aebb220909.png)


Suggested:
![image](https://user-images.githubusercontent.com/63015754/191614796-084b0f7d-64bc-497e-ae51-2073ed0446cd.png)


Also, not sold on the blue, but it's fine :)

Reply:
@CobyPear @enZane Agree that the gradient change makes this pop a little bit more. The blue is a little boring, but the previous gradient was a little ugly, so I'll take this :)